modules/appeals_api/app/sidekiq/appeals_api/notice_of_disagreement_clean_up_pii.rb

```diff
@@ -3,7 +3,7 @@
 require 'sidekiq'
 
 module AppealsApi
-  class NoticeOfDisagreementCleanUpWeekOldPii
```

Comment:
I didn't update the Supplemental Claim job because it was already titled `SupplementalClaimCleanUpPii`